contracts/NFTFloorPriceConsumerV3.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.7;
+
+import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
+
+contract NFTFloorPriceConsumerV3 {
```

Comment:
Do we need this in here? It's just another price feed. Have people been asking for this?